messaging/relayer/cmd/admin/main.go

```diff
@@ -69,15 +69,10 @@ func main() {
 					{
 						Name:      "stop",
 						Usage:     "Stop observing contract",
-						ArgsUsage: "[url chainID contractAddr]",
+						ArgsUsage: "[url]",
 						Action: func(c *cli.Context) error {
 							url := c.Args().Get(0)
-							chainID, err := strconv.ParseUint(c.Args().Get(1), 10, 64)
-							if err != nil {
```

Comment:
yes. I was referring to the chain ID definition on Ethereum. I guess string makes more sense as we could support chains that are not ethereum based.

Reply:
This would require some work. I'll do it in a separate PR.

Reply:
We have discussed this off-github. My thoughts are:
- The Java SDK code currently mixes blockckhain IDs and chain IDs, and treats them as interchangable. This will need to be fixed.
- The blockchain ID is a 256 bit number. 
- Maybe have a struct / type in the code that is the EIP 3220 blockchain ID, and from it the chain Id and the blockchain ID in various formats can be extracted.